make/modules/java.desktop/Lib.gmk

```diff
@@ -38,6 +38,7 @@ include lib/AwtLibraries.gmk
 include lib/ClientLibraries.gmk
 
 ifeq ($(call isTargetOs, aix), false)
+ifneq ($(OPENJDK_TARGET_OS_ENV), bsd.openbsd)
```

Comment:
You should introduce a new `isTargetOsEnv`, in the similar vein as the other macros. Then you can do:
```
ifeq ($(call isTargetOs, aix)+$(call isTargetOsEnv, bsd.openbsd), false+false)
```

to combine this into a single condition. (The `+` trick is something we use elsewhere to overcome the lack of proper boolean operators in make syntax)

Otherwise, you will need to change the indentation level of everything between the new if/endif. Which I recommend against.

Reply:
That's a good call. I've noticed the `+` trick elsewhere, so that is familiar.